test/refactoring/SyntacticRename/FindRangeOutputs/property-wrapper-init/body.swift.expected

```diff
@@ -0,0 +1,37 @@
+public struct Outer {
+    @propertyWrapper
+    public struct InnerWrapper<T> {
+        public var value: T
+        public /*init:def*/init(initialValue: T) {
```

Comment:
Do you have a test somewhere for `init(initialValue: T, other: T)` with `@Wrapper(other: 1) var foo: Int = 2`?

Reply:
Oh thanks – I didn't realize that was in yet. I think the initializer rename should still be fine, but given the passed arguments could be closure bodies and can now be split between the custom attribute and the var initializer, NameMatcher and SyntaxModel may have issues with things being out of source order...

Reply:
Added tests and it looks like they're actually fine. They're running over the non-typechecked AST, so they're still two distinct pieces.